tests/test_proxy.py

```diff
@@ -130,11 +130,17 @@ def test_seed_classmethod(self):
 
     def test_seed_class_locales(self):
         Faker.seed(2043)
+        count = 5
         fake = Faker(["en_GB", "fr_FR", "en_IN"])
-        name = fake.name()
+        first_list = [fake.name() for _ in range(count)]
+        assert len(set(first_list)) == count
```

Comment:
what's the reason for casting to a `set`?

Reply:
got it 👍🏼  Would you add a comment above the assertion?

Reply:
done